src/components/ResultPanel/ResultPanel.css

```diff
@@ -2,6 +2,7 @@
 
 .recordType {
   display: inline-block;
+  flex-shrink: 0;
```

Comment:
Let's have above the display. BTW, do we need display there as it seems to be flex item?

Reply:
Done.